core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -183,16 +184,19 @@ public Boolean apply(Void input) {
 
   @Override
   public void start() throws Exception {
+    LOG.info("Starting Alluxio master @{}", mRpcAddress);
```

Comment:
yeah, the integration test won't have this. maybe it is more scalable to add  this logging here. I've removed this and added corresponding logging in the local clusters.